src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,20 @@ public function __toString(): string
     {
         return (string) $this->value;
     }
+
+    public static function fromBitfield(int $value, array $values): self
```

Comment:
I suggest adding a phpdoc `array<int, string>` for `$values` to document the expected signature

Reply:
updated